src/pages/Governance/ProposalList/GovernanceProposal/index.tsx

```diff
@@ -176,24 +208,21 @@ const GovernanceProposalUi: React.FC<GovernanceProposalProps> = ({
         )
       }
       footer={
-        endDate && proposalState === 'Active' ? (
+        statusDate && statusKey ? (
           <div css={styles.timestamp}>
             <Typography variant="small2">
-              <div css={styles.greenPulseContainer}>
-                <GreenPulse css={styles.greenPulse} />
-              </div>
```

Comment:
Let's keep that in, when the proposal is active only. It's my bad though, the designs I linked haven't been updated to include this animation which was added later on. I've asked Pavel to fix that in the designs so there's no confusion

Reply:
Done!